### PR TITLE
Add forms 6 and 60 to datasources

### PR DESCRIPTION
**Pull request by zschira**
# Background
This pull request adds FERC Forms 6 and 60 datasources. It does not integrate these forms into PUDL in any way, but it will enable us to archive them on Zenodo. This will enable the use of the [FERC XBRL Extractor](https://github.com/catalyst-cooperative/ferc-xbrl-extractor/tree/main/src/ferc_xbrl_extractor) to create raw SQLite databases containing data extracted from filings of these forms.

### CODE REVIEW

File: src/pudl/metadata/sources.py [hunk 1]
```diff
@@ -330,6 +330,36 @@
...
+    },
+    "ferc60": {
+        "title": "FERC Form 60",
+        "path": "https://www.ferc.gov/general-information-1/oil-industry-forms/form-6-6q-historical-vfp-data",
+        "description": (
+            "The Federal Energy Regulatory Commission (FERC) Form 60 is a "
+            "comprehensive financial and operating report submitted for centralized "
+            "centralized service companies."
+        ),
+        "field_namespace": "ferc6",
```
Comment (zaneselvans) [inline src/pudl/metadata/sources.py:356]: Should probably be `ferc60` here.

File: src/pudl/metadata/sources.py [hunk 2]
```diff
@@ -330,6 +330,36 @@
...
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+    },
+    "ferc60": {
+        "title": "FERC Form 60",
+        "path": "https://www.ferc.gov/general-information-1/oil-industry-forms/form-6-6q-historical-vfp-data",
+        "description": (
+            "The Federal Energy Regulatory Commission (FERC) Form 60 is a "
+            "comprehensive financial and operating report submitted for centralized "
+            "centralized service companies."
```
Comment (zaneselvans) [inline src/pudl/metadata/sources.py:354]: double centralized.

File: src/pudl/metadata/sources.py (hunk 1 above)
Comment (zschira) [reply src/pudl/metadata/sources.py]: Fixed

File: src/pudl/metadata/sources.py (hunk 2 above)
Comment (zschira) [reply src/pudl/metadata/sources.py]: Fixed

**Comment by zschira** [comment]
Closing, these changes are included in `ferc_datasette` branch under review now

**Pull request closed without merging**